docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -490,8 +490,10 @@ iceberg.catalog.type=rest
 iceberg.rest-catalog.uri=http://iceberg-with-rest:8181
 ```
 
-The REST catalog does not support [view management](sql-view-management) or
-[materialized view management](sql-materialized-view-management).
+The REST catalog supports [view management](sql-view-management) 
```

Comment:
Can you confirm that all these SQL command for view management in the linked section are supported please

Reply:
Yes, all the SQL commands in the linked section are supported.